src/components/PageSection.tsx

```diff
@@ -4,7 +4,11 @@ type Props = {
   title: string
   description: string
   textPosition: 'left' | 'right'
-  image: string
+  image?: string
+} & typeof defaultProps
```

Comment:
not sure abt this practice but seems unnecessary cos image is alr explicitly typed as string